assets/scss/_k8s_nav.scss

```diff
@@ -0,0 +1,160 @@
+.td-navbar {
+  background: transparent !important;
+  position: fixed !important;
+  transition: 0.3s;
+  width: 100%;
+
+  // Based on from Bootstrap's _navbar.scss
+  .active > .nav-link,
+  .nav-link.active {
+    box-shadow: 0 2px $navbar-dark-active-color;
+  }
+
+  .navbar-brand__name {
+    display: none;
+  }
+
+  .navbar-brand {
+    @include media-breakpoint-down(sm) {
+      margin-left: auto;
+    }
+
+    svg {
+      height: 38px;
+    }
+  }
+
+  @media only screen and (max-width: 1500px) {
+    .td-navbar-nav-scroll {
+      font-size: .875rem;
+    }
+  }
+
+  #hamburger {
+    color: $navbar-dark-color;
+
+    @include hover-focus() {
+      color: $navbar-dark-hover-color;
+    }
+
+    margin-left: auto;
+    font-size: xx-large;
+
+    // Bootstrap buttons have a noticeable depression effect when clicked which is not desirable for the hamburger button
+    &:focus,
+    &:active {
+      box-shadow: none !important;
+    }
+  }
+}
+
+.navbar-bg-onscroll {
+  background: white !important;
+  box-shadow: 0 1px 2px $medium-grey;
+
+  .navbar-nav .nav-link, #hamburger {
+    color: $dark-grey;
+
+    @include hover-focus() {
+      color: $medium-grey;
+    }
+  }
+
+  .navbar-nav {
+    // Based on from Bootstrap's _navbar.scss
+    .show > .nav-link,
+    .nav-link.show {
+      color: $medium-light-grey;
+    }
+
+    .active > .nav-link,
+    .nav-link.active {
+      color: $dark-grey;
+      box-shadow: 0 2px $dark-grey;
+    }
+  }
+}
+
+/* SIDE-DRAWER MENU */
```

Comment:
Is this right, given the filename?

Reply:
Maybe mention that it's for small viewports or mobile devices. There's no context to make that obvious.